content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -13,23 +13,56 @@ This is a small example application we will monitor in this guide.
 
 Create an empty package.json:
 
-```sh
+{{< tabs TypeScript JavaScript >}}
+
+{{< tab >}}
+tsc --init
+{{< /tab >}}
+
+{{< tab >}}
 npm init -f
-```
+{{< /tab >}}
+
+{{< /tabs >}}
```

Comment:
AFAIK, you'll need `npm init -f` for TS or JS, so you'll want to show the command as it was before, not inside tabs.

Also, this is on a section entitled **Dependencies**. Again, AFAIK of what `tsc --init` does, this command should be in the next section, not this section on dependencies.

Reply:
Oh thank you so much. I have done the requested changes. I have kept single tab for typescript `tsc --init` command to make it stand out like other commands